### PR TITLE
WEB-1362: Add CCX Coach button to the top of the list.

### PR DESCRIPTION
**Pull request by lukyfebrianto**
**Description:** Add CCX Coach button to the top of the list

**JIRA:** https://liv-it.atlassian.net/browse/WEB-1362

**Merge deadline:** List merge deadline (if any)

**Configuration instructions:** List any non-trivial configuration
instructions (if any).

**Testing instructions:**

1. Open page A
2. Do thing B
3. Expect C to happen
4. If D happened instead - check failed.

**Reviewers:**
- [x] tag reviewer
- [x] tag reviewer

**Merge checklist:**
- [x] All reviewers approved
- [x] CI build is green
- [ ] Documentation updated (not only docstrings)
- [x] Commits are squashed

**Post merge:**
- [ ] Delete working branch (if not needed anymore)

**Author concerns:** List any concerns about this PR - inelegant
solutions, hacks, quick-and-dirty implementations, concerns about
migrations, etc.

### CODE REVIEW

**Comment by lukyfebrianto** [comment]
Moving of the Add CCX Coach button to the top of the list.
screenshot from 2017-11-01 12-35-01

File: lms/templates/instructor/instructor_dashboard_2/membership.html [hunk 1]
```diff
@@ -23,13 +33,6 @@
         <tbody></tbody>
       </table>
     </div>
-    <div class="bottom-bar">
-      <label>
-          <span class="label-text sr">{{add_placeholder}}</span>
-          <input type="text" id="add-field" name="add-field" class="add-field" placeholder="{{add_placeholder}}">
-      </label>
-      <input type="button" name="add" class="add" value="{{add_btn_label}}">
-    </div>
```
Comment (polesye) [inline lms/templates/instructor/instructor_dashboard_2/membership.html]: @lukyfebrianto sorry for the bad descriptions, but could you leave the button at the bottom of the list and add one more at the top?

Comment (lukyfebrianto) [reply lms/templates/instructor/instructor_dashboard_2/membership.html]: @polesye  so we have 2 button(top and bottom) right ?

Comment (polesye) [reply lms/templates/instructor/instructor_dashboard_2/membership.html]: yes

**Review by polesye — APPROVED** [review]
looks good to merge after addressing my comment

File: lms/templates/instructor/instructor_dashboard_2/membership.html (hunk 1 above)
Comment (TamoshaytisV) [reply lms/templates/instructor/instructor_dashboard_2/membership.html]: Agree to have 2 buttons 'cause as i remember edx do not have pagination for emails list

**Review by TamoshaytisV — APPROVED** [review]
Please address Anton's comment

File: lms/templates/instructor/instructor_dashboard_2/membership.html (hunk 1 above)
Comment (lukyfebrianto) [reply lms/templates/instructor/instructor_dashboard_2/membership.html]: done @polesye , @TamoshaytisV 
screenshot from 2017-11-02 11-10-48

**Review by polesye — APPROVED** [review]

**Pull request merged**